src/ReactiveUI/ReactiveUI.csproj

```diff
@@ -6,8 +6,6 @@
     <RootNamespace>ReactiveUI</RootNamespace>
     <PackageDescription>A MVVM framework that integrates with the Reactive Extensions for .NET to create elegant, testable User Interfaces that run on any mobile or desktop platform. This is the base package with the base platform implementations</PackageDescription>
     <PackageId>ReactiveUI</PackageId>
-    <Nullable>enable</Nullable>
```

Comment:
Was this a mistake?

Reply:
No intentional, they are automatic from the solution props